docs/_static/uml/core-templates-read.uml

```diff
@@ -0,0 +1,24 @@
+    @startuml
+    hide footbox
+    skinparam shadowing false
+
+    actor Client
+    boundary Service
+    box "Service Cache"
+        database "Filesystem Cache" as Filesystem
+        database "Memory Cache" as Cache
+    end box
+    boundary Worker
+    entity "Remote Git" as Git
+    entity "External Templates Repository" as External
+
+    == Read manifest from a templates repository ==
+    Client->Service: GET /<prefix>/templates.read_manifest
+    activate Service
+
+    Service->External: Clone templates repository
+    External->Filesystem: Persist project
```

Comment:
What does `persist project` mean in this context?

Reply:
I took inspiration from the description in the other charts.
It means that the project is saved to the local Filesystem cache so that it can be potentially used for other operations -- e.g. creating another project for the same user almost at the same time